src/app/public/modules/paging/paging.component.ts

```diff
@@ -15,28 +15,52 @@ import {
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SkyPagingComponent implements OnChanges {
+
+  /**
+   * Specifies the page number of the currently displayed page. Page numbers start at 1 and increment.
+   */
   @Input()
-  public pageSize: number = 10;
+  public currentPage: number = 1;
 
+  /**
+   * Specifies the total number of items across all pages.
+   */
   @Input()
-  public maxPages: number = 5;
+  public itemCount: number = 0;
 
+  /**
+   * Specifies the maximum number of pages to display in the pagination control.
+   */
   @Input()
-  public currentPage: number = 1;
+  public maxPages: number = 5;
 
+  /**
+   * Specifies the number of items to display per page.
+   */
   @Input()
-  public itemCount: number = 0;
+  public pageSize: number = 10;
 
+  /**
+   * Specifies a label for the pagination control to support
+   * [accessibility](https://developer.blackbaud.com/skyux/components/paging#accessibility).
```

Comment:
We don't want this link anymore. It points to a section on the existing page that will be gone when we go live with these docs. The content here already overlaps the content in that Accessibility section, so I'd just remove the link and then pull in the little bit of info that we are linking to that isn't already included. Here's what I'm thinking:

```Specifies a label for the pagination control to support accessibility when an application includes multiple paging components on the same page. The label should be unique and descriptive to help users of assistive technology differentiate pagination controls and understand what each one does.```

Reply:
Do you think we need "to support accessibility"? Seems redundant with the other line. I'll update it but let me know if you disagree.

Reply:
Works for me.